packages/bot-web-ui/src/components/load-modal/local.jsx

```diff
@@ -58,15 +59,17 @@ const LocalComponent = ({ handleFileChange, is_mobile, loaded_local_file, setLoa
                         <React.Fragment>
                             <Icon icon={'IcPc'} className='load-strategy__local-icon' size={is_mobile ? 96 : 128} />
                             <div className='load-strategy__local-title'>
-                                <Localize i18n_default_text='Drag your file here' />
+                                <Localize i18n_default_text='Drag your XML file here' />
                             </div>
                             <div className='load-strategy__local-description'>
                                 <Localize i18n_default_text='or, if you prefer...' />
                             </div>
                         </React.Fragment>
                     )}
                     <Button
-                        text={localize('Select a file from your device')}
+                        text={localize(
+                            is_file_supported ? 'Select an XML file from your device' : 'Please upload an XML file'
+                        )}
```

Comment:
This won't work as the regex pattern (we use to find strings to localize) isn't picking up on the string. The ternary should be placed outside the `localize` call, e.g.

```JS
is_file_supported 
    ? localize('Select an XML file from your device')
    : localize('Please upload an XML file')
```